torchvision/csrc/image/readpng.h

```diff
@@ -0,0 +1,14 @@
+#ifndef READPNG_H
+#define READPNG_H
+
+#include <torch/torch.h>
```

Comment:
I was also thinking that we should maybe move this folder inside `csrc/cpu`, because the decoding happens on the CPU, and we could potentially add GPU decoding in the future. Thoughts?

Reply:
You are right, I'll do it.

Reply:
I've moved the code. 

I just left an  `image.h` header file in `csrc` to serve as an entry point for the image features. 

I moved the actual implementation in `csrc/cpu` and remove the namespaces. I looked at the code base a bit and I have seen that namespaces are not used in the code base. I removed it to keep the consistency of the code.
 
Thanks